app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] A comment should belong to an `author`, not a `user`. So you should have this instead:

  ```
  belongs_to :author, class_name: 'User'
  ```
 
- [ ] I can also see from your database schema that what you have is `user_id` in the likes table. You need to update your references to point to the `author`, instead of the `user`. Refer to this comment for an idea of how to do that: [https://stackoverflow.com/a/42056089](https://stackoverflow.com/a/42056089)